appveyor.yml

```diff
@@ -52,7 +53,8 @@ for:
 
   # stack is not able to build test dependencies with lts-6
```

Comment:
This comment seems outdated now. We still don't test `dhall-json` with lts-6 due to dependencies though?!